pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```
@@ -131,7 +131,7 @@ spec:
     - name: output
       workspace: build-workspace-pv
     - name: ssl-ca-directory
-      workspace: ssl-certs
+      workspace: git-ssl-certs
```

```suggestion
      workspace: git-tls-certs
```

Also, should it be `-certs` or `-cert` now?

---

We should keep this as `git-ssl-*` to maintain clarity between dependencies (boto3, aws s3, git, ...) that still publicly reference ssl within their respective tools (`use_ssl`, `--no-verify-ssl`, `http.sslVerify`, ...)

---

Yes SSL is still used even though I agree TLS is better. I changed to not be plural though 👍